loleaflet/css/notebookbar.css

```diff
@@ -79,7 +80,7 @@
 	display: inline-block;
 	position: relative;
 	z-index: 1000;
-	height: 100%;
+	height: 32px;
```

Comment:
I don't think this change is necessary. Can you please check and if it's not remove it from the commit?

Reply:
checked and isn't necessary.